MapboxNavigation/NavigationViewController.swift

```diff
@@ -190,61 +197,81 @@ public protocol NavigationViewControllerDelegate: VisualInstructionDelegate {
  
  - seealso: CarPlayNavigationViewController
  */
+@objcMembers
```

Comment:
I never knew about `@objcMembers`! Is the benefit that it saves you some keyboard typing?

Does it preclude us from using swift-only types in private methods?

Reply:
Ah, I missed https://github.com/maplibre/maplibre-navigation-ios/pull/54#discussion_r1629649939 - seems like it's primarily about saving some keyboard typing. Fine by me.